<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_classes_dialog</class>
 <widget class="QDialog" name="student_classes_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What classes are taken by</string>
   </property>
  </widget>
  <widget class="QComboBox" name="student_menu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="query_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do query</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="class_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>321</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
